paper/paper.md

```diff
@@ -0,0 +1,64 @@
+  - neuroscience
+  - neuroimaging
+  - eeg
+  - microstates
+  - brain
+authors:
+  - name: Victor Férat^[Co-first author]^[Corresponding author] # note this makes a footnote saying 'Co-first author'
+    orcid: 0000-0003-1952-7657
+    affiliation: 1
+  - name: Mathieu Scheltienne^[Co-first author] # note this makes a footnote saying 'Co-first author'
+    orcid: 0000-0001-8316-7436
+    affiliation: 2
+  - name: Denis Brunet
+    affiliation: "1, 3" # (Multiple affiliations must be quoted)
+  - name: Tomas Ros
+    orcid: 0000-0001-6952-0459
+    affiliation: "1, 3" # (Multiple affiliations must be quoted)
+  - name: Christoph Michel
+    orcid: 0000-0003-3426-5739
+    affiliation: "1, 3" # (Multiple affiliations must be quoted)
+affiliations:
+ - name: Functional Brain Mapping Laboratory, Department of Basic Neurosciences, Campus Biotech, University of Geneva, Geneva, Switzerland
+   index: 1
+ - name: Human Neuroscience Platform, Fondation Campus Biotech Geneva, Geneva, Switzerland
+   index: 2
+ - name: Centre for Biomedical Imaging (CIBM) Lausanne-Geneva, Geneva, Switzerland
+   index: 3
+date: 30 May 2022
+bibliography: paper.bib
+---
+
+# Summary
+Microstate analysis of the electroencephalogram (EEG), introduced in 1972 by Lehman [citation] is a spatiotemporal analysis technique that takes advantage of the full spatial resolution of EEG recordings. Formalized by Pascual Marqui and colleagues [@pascual-marqui_segmentation_1995], it consists in studying the distribution of the surface EEG potential maps over time and thus transforming the EEG recordings into sequences of successive states of variable duration called EEG microstates.
+
+
+Pycrostates implements multiple modules that allow researchers to apply this method to their data:
+
+- the dataset module provides direct access to preprocessed EEG data from the LEMON dataset [@babayan_mind-brain-body_2019] and thus offers test and experimental data.
+the preprocessing module completes the classical EEG preprocessing tools [@pernet2020issues] with those specific to the preparation of data for microstate analysis.
+
+- a segmentation module allows the sequences of microstates obtained to be studied
+
+- the cluster module gathers the different algorithms of transformation of the recordings in sequence of microstates. It takes example of the API of scikit-learn [@pedregosa_scikit-learn_nodate] [@buitinck_api_2013] for a simple and effective integration.
+
+- a segmentation module that allows to study the sequences of microstates obtained from clustering prediction.
+
+- a metrics module that allows to quantify the quality of the clustering results.
+
+By construction, Pycrostates is evolutionary. Future improvements can easily be added, such as the addition of clustering algorithms or new tools for sequence analysis such as Markov chains. Authors welcome proposals for improvements and new contributions and look forward to promoting and advancing the analysis of EEG microstates.
```

Comment:
1. I think I would rank by importance here, starting with `dataset` did feel weird.
2. Suggestion of how I would formulate it, feel free to change, it's just a proposition!

Don't forget that the 'no-repetition' is only a French rule, if a word/syntax has to be repeated 10 times, it's fine in English ;)

```suggestion
- the cluster module supports the different algorithms that transform the signals into sequences of microstates. 
- the segmentation module supports the study of microstates sequences.
- the metrics module quantifies the quality of the fitted clustering algorithms.

Pycrostates is supported by additionals modules to help researchers in the development of analysis pipelines:
- the dataset module provides direct access to preprocessed data that can be used to test pipelines. As of writing, it supports the LEMON dataset [@babayan_mind-brain-body_2019] comprising 227 healthy participants' EEG recordings.
- the viz module provides visualization tools to perform microstates analysis.

By design, Pycrostates supports natively all data types from MNE-Python [@Agramfort2013] and is not restricted to EEG data. It is designed to support future improvements and additions such as different clustering algorithms or new tools for sequence analysis suck as Markov chains. Moreover, this work builds its API on top of the robust MNE-Python ecosystem, enabling a seamless integration microstates analysis.
```

The sentence `Authors welcome proposals for improvements and new contributions and look forward to promoting and advancing the analysis of EEG microstates.` feels weird here, probably at the end of the statement of need IMO.